### PR TITLE
Add base.html

### PR DESCRIPTION
**Pull request by hirundinidae**

### CODE REVIEW

**Review by skyflaren — APPROVED** [review]

File: core/templates/core/base.html [hunk 1]
```diff
@@ -3,53 +3,111 @@
...
+            <li class="nav-item">
+                <a class="nav-link" href="{{ items }}">{{ heading }}</a>
+            </li>
+            {% elif items|gettype == 'dict' %}
+            <li>
+                <a class="dropdown-trigger" href="#!" data-target="dropdown1">{{ heading }}<i class="material-icons right">arrow_drop_down</i></a>
+                <ul id="dropdown1" class="dropdown-content">
+                    {% for item, path in items.items %}
+                        <li>
+                            <a href="#!">{{ item }}</a>
```
Comment (fluix-dev) [inline core/templates/core/base.html:68]: Should this be
```suggestion
                            <a href="{{ path }}">{{ item }}</a>
```
? You never use `path`.

**Review by fluix-dev — COMMENTED** [review]
There's inconsistent indentation for the template blocks like `{% if %}` and `{% for %}`. I would suggest indenting the following lines.

File: core/templates/core/base.html [hunk 2]
```diff
@@ -3,53 +3,111 @@
...
+    <li><a href="{% url 'profile_redirect' %}">Profile</a></li>
+    {% if user.is_staff %}
+    <li><a href="{% url 'admin:index' %}">Admin</a></li>
+    {% endif %}
+    <li><a href="{% url 'account_logout' %}">Logout</a></li>
+    {% else %}
+    <li><a href="{% url 'account_login' %}?next={{ request.get_full_path }}">Login</a></li>
+    <li><a href="{% url 'account_signup' %}?next={{ request.get_full_path }}">Register</a></li>
+    {% endif %}
+    <li><a>hi</a></li>
```
Comment (fluix-dev) [inline core/templates/core/base.html:17]: ?

File: core/templates/core/base.html [hunk 3]
```diff
@@ -3,53 +3,111 @@
...
+</ul>
+<ul class="sidenav" id="mobile-demo">
+    {% settings_value "NAVBAR" as navbar %}
+    {% for heading, items in navbar.items %}
+    {% if items|gettype == 'str' %}
+    <li >
```
Comment (fluix-dev) [inline core/templates/core/base.html:23]: Extra space.

File: core/templates/core/base.html [hunk 4]
```diff
@@ -3,53 +3,111 @@
...
+        <a href="{{ items }}">{{ heading }}</a>
+    </li>
+    {% elif items|gettype == 'dict' %}
+    <li class="divider"></li>
+    <li>
+        {% for item, path in items.items %}
+            <li>
+                <a href="#!">{{ item }}</a>
```
Comment (fluix-dev) [inline core/templates/core/base.html:31]: See similar comment below.

File: core/templates/core/base.html [hunk 5]
```diff
@@ -3,53 +3,111 @@
...
+            </li>
         {% endfor %}
-        <li class="nav-item dropdown">
-            <a class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
+    </li>
+    {% else %}
+    {{ items|gettype }}
```
Comment (fluix-dev) [inline core/templates/core/base.html:36]: You probably don't need this clause, though maybe useful for debugging?

File: core/templates/core/base.html [hunk 6]
```diff
@@ -3,53 +3,111 @@
...
+    <li><a href="{% url 'account_logout' %}">Logout</a></li>
+    {% else %}
+    <li><a href="{% url 'account_login' %}?next={{ request.get_full_path }}">Login</a></li>
+    <li><a href="{% url 'account_signup' %}?next={{ request.get_full_path }}">Register</a></li>
+    {% endif %}
+</ul>
+<nav>
+    <div class="nav-wrapper">
+        <a class="brand-logo" href="{% url 'index' %}">Metropolis</a>
+        <a href="#" data-target="mobile-demo" class="sidenav-trigger"><i class="material-icons">menu</i></a>
```
Comment (fluix-dev) [inline core/templates/core/base.html:54]:
```suggestion
        <a href="#" data-target="mobile-demo" class="sidenav-trigger" aria-label="Open navigation menu"><i class="material-icons" aria-hidden="true">menu</i></a>
```

You should do similar for other uses of interactive icons. For all icons in general, I like to follow [this guide from FontAwesome on accessibility](https://fontawesome.com/v4.7/accessibility/).

File: core/templates/core/base.html (hunk 1 above)
Comment (skyflaren) [reply core/templates/core/base.html:68]: yes - missed it since we don't have any right now, good catch

File: core/templates/core/base.html (hunk 2 above)
Comment (skyflaren) [reply core/templates/core/base.html:17]: thought I got rid of that earlier - debug code, meant to be deleted

File: core/templates/core/base.html (hunk 6 above)
Comment (skyflaren) [reply core/templates/core/base.html:54]: ty for the guide - will keep in mind

**Pull request merged**